spec/features/admin/users/platform_admin_can_delete_a_user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
```

Comment:
this doesn't feel like a feature test to me---should this test maybe be updated to check what the user sees, instead of testing what the database holds?